ChatAPI.Application/UseCases/Abstractions/IUserRepository.cs

```diff
@@ -7,7 +7,7 @@ public interface IUserRepository
 	{
 		void AddUserDevice(UserRegisterDTO dto);
 		Task<User?> GetUser(string phone);
-		Task<User?> GetUserWithUserDevicesIncluded(string phone);
+		Task<User?> GetUser_DevicesAndLoginCodeIncluded(string phone);
```

Comment:
I prefer not to have underlines in the method names. You can say GetUserIncludingDevicesAndLoginCode